app/models.py

```diff
@@ -1358,8 +1358,8 @@ def serialize(self, with_users=False):
                 'withdrawnAt': self.withdrawn_at.strftime(DATETIME_FORMAT)
             })
 
-        if self.copied_from_brief_id:
-            data.update({'copiedFromBriefId': self.copied_from_brief_id})
+        if self.is_a_copy:
```

Comment:
Do we want this to be conditional?  Seems like having the key always there with a true/false value would be fine?